data.sql

```diff
@@ -4,3 +4,9 @@ INSERT INTO animals VALUES (1, 'Agumon', cast('2020/02/03' as date), 0, '1', 10.
 INSERT INTO animals VALUES (2, 'Gabumon', cast('2018/11/15' as date), 2, '1', 8);
 INSERT INTO animals VALUES (3, 'Pikachu', cast('2021/01/7' as date), 1, '0', 15.04);
 INSERT INTO animals VALUES (4, 'Devimon', cast('2017/06/12' as date), 5, '1', 11);
+INSERT INTO animals VALUES (5, 'Charmander', '2020/02/08', 0, '1', -11);
+INSERT INTO animals VALUES (6, 'Plantmon', '2022/11/15', 2, '1', -5.7);
+INSERT INTO animals VALUES (7, 'Squirtie', '1993/04/02', 3, '0', 12.13);
```

Comment:
- [OPTIONAL] The weight for this animal should be a negative value `-12.13` but you inserted a positive one `12.13`. Updating it will be a great move for you not to get errors in your future queries.